rpcs3/Emu/RSX/GL/GLFragmentProgram.cpp

```diff
@@ -2,7 +2,6 @@
 #include "GLFragmentProgram.h"
 
 #include "Emu/system_config.h"
-#include "GLFragmentProgram.h"
```

Comment:
lol. this doesn't sound right. But if it compiles in VS then there must be some circular dependency right??

Reply:
ah. it was a double include